migration/README.md

```diff
@@ -1,4 +1,12 @@
-# Running Migrator CLI
+There are two commands in this crate which assist with running database migrations.
+
+All commands require that you either export `DATABASE_URL` or provide a `--database-url`
+flag, containing a valid [PostgreSQL connection string](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING)
+(e.g. `postgres://username:password@hostname:port/database`).
+
```

Comment:
nit: to avoid the long line, you could use this form:
```suggestion
flag, containing a valid [PostgreSQL connection string][pgsql-conn-str]
(e.g. `postgres://username:password@hostname:port/database`).

[pgsql-conn-str]: https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING
```